src/main/resources/resourcepacks/dungeons/pack.mcmeta

```diff
@@ -1,6 +1,6 @@
 {
   "pack": {
-    "pack_format": 8,
-    "description": "Sounds from Dungeons for 1.5.0+mc1.18"
+    "pack_format": 9,
+    "description": "Sounds from Dungeons for 1.5.0+mc1.19"
```

Comment:
I think the resource pack has some superfluous entries now, but at least it seems to be working.